Controller/ResourceController.php

```diff
@@ -123,17 +123,38 @@ protected function findOrThrowNotFound(ResourceInterface $resource, $id)
     protected function getCollectionData(ResourceInterface $resource, Request $request)
     {
         $page = (int) $request->get('page', 1);
-
+        $order = [];
         $filters = [];
-        foreach ($resource->getFilters() as $resourceFilter) {
+        $resourceFilters = $resource->getFilters();
+
+        foreach ($resourceFilters as $resourceFilter) {
             if (null !== $value = $request->get($resourceFilter['name'])) {
                 $resourceFilter['value'] = $value;
                 $filters[] = $resourceFilter;
             }
         }
 
+        // Add order filters
+        $requestOrderFilters = $request->get('order');
+        foreach ($requestOrderFilters as $key => $value) {
```

Comment:
Not sur if it's relevant for now. As it's highly controller related, I think we can let this method here. We will extract it in a trait or something like that if it grows.